database/migrations/2020_08_18_032818_add_member_character_slug.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddMemberCharacterSlug extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('characters', function (Blueprint $table) {
+            $table->string('slug', 50)->after('name')->index();
+        });
+
+        Schema::table('members', function (Blueprint $table) {
+            $table->string('slug', 50)->after('username')->index();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('characters', function (Blueprint $table) {
+            //
+        });
+
+        Schema::table('members', function (Blueprint $table) {
+            //
+        });
```

Comment:
Just a note that these don't do anything unless we program them to undo the changes we made. (but we can skip it for now)

`php artisan migrate:rollback` can then be run to undo a migration.

Reply:
yeah, I was just stubbing it for later if we need.